tests/_bootstrap.php

```diff
@@ -4,7 +4,7 @@
 
 defined('YII_TEST_ENTRY_URL') or define('YII_TEST_ENTRY_URL', parse_url(\Codeception\Configuration::config()['config']['test_entry_url'], PHP_URL_PATH));
 defined('YII_TEST_ENTRY_FILE') or define('YII_TEST_ENTRY_FILE', dirname(dirname(__DIR__)) . '/web/index-test.php');
-defined('VENDOR_DIR') or define('VENDOR_DIR', __DIR__ . '/../vendor');
+defined('VENDOR_DIR') or define('VENDOR_DIR', __DIR__ . '/../../../../vendor');
```

Comment:
This is not correct. The module has its own vendor dir which should be used for testing. The travis tests rely on this and may be this is the reason the tests are failing.